utils/cursor_md_exporter.py

```
@@ -0,0 +1,144 @@
+    tool_breakdown: dict[str, int] = {}
+    for b in bubbles:
+        tcs = (b.get("metadata") or {}).get("toolCalls") or []
+        total_tool_calls += len(tcs)
+        for tc in tcs:
+            tn = tc.get("name", "unknown")
+            tool_breakdown[tn] = tool_breakdown.get(tn, 0) + 1
+
+    # Frontmatter.
+    fm_lines = ["---"]
+    fm_lines.append(f"log_id: {session_id}")
+    fm_lines.append(f"log_type: cli_agent")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)